actions-job.php

```diff
@@ -1,4 +1,34 @@
- <?php include_once('header.php'); // Include Header once ?>
+ <?php include_once('header.php'); // Include Header once
+ 
+ if (isset($_POST['new_job'])) {
+     if ($_SESSION['is_company'] = true) { // Check if the log is company
+    // initializing variables
+         var_dump($_SESSION['is_company']);
```

Comment:
You missed a var_dump here. A good practice is to always check the code changes you did before you commit them.

I always CTRL+F search for var_dump before committing.